salt/defaults/events.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+'''
+Event constants used by listeners and servers, to be imported elsewhere in Salt code.
+
+Do NOT, import any salt modules (salt.utils, salt.config, etc.) into this file,
+as this may result in circular imports.
+
+.. versionadded:: Neon
+'''
+
+# Constants for events on the minion bus
+EV_PILLAR_COMPLETE = '/salt/minion/minion_pillar_complete'
+EV_MOD_COMPLETE = '/salt/minion/minion_mod_complete'
```

Comment:
Sorry being so much annoying here, but this is already in `events`, so no need to repeat yourself with `EV_` prefix. However, `MINION_` would help here better. :wink: You will be using it like:

```python
if salt.defaults.events.MINION_PILLAR_COMPLETE .....
```

This is perfectly readable & comprehensible.

Reply:
@isbm, I adapted the naming as requested.